src/RestWrite.js

```diff
@@ -1195,9 +1195,12 @@ RestWrite.prototype._updateResponseWithData = function(response, data) {
   const clientSupportsDelete = ClientSDK.supportsForwardDelete(this.clientSDK);
   this.storage.fieldsChangedByTrigger.forEach(fieldName => {
     const dataValue = data[fieldName];
-    const responseValue = response[fieldName];
 
-    response[fieldName] = responseValue || dataValue;
+    if(response.hasOwnProperty(fieldName)) {
```

Comment:
Just inverse the if, and set dataValue if response don’t have the value. Right?

Reply:
Oh yeah that's a redundant set for sure...I just removed it.